### DISCUSSION TITLE
ESLint disable not working.

### DISCUSSION THREAD

**Discussion by jwerre** [eslint/eslint#17489] [Q&A] [answered]
I ran into this issue where I'm trying to use the [global `import` function](https://nodejs.org/docs/latest-v18.x/api/esm.html#import-expressions) along side my CommonJS `require` statements and ESLint is yelling at me.

```js
const AWS = require('aws-sdk');
const { get } = require('lodash');
/* eslint-disable */
const someES6Module = import('es6-module'); // eslint-disable-line
/* eslint-enable */
```

```
12:24  error  Parsing error: Unexpected token import
✖ 1 problem (1 error, 0 warnings)
```

I'm only using it once and it's temporary until I can find the time to migrate to ES6 modules so I'd just like to ignore the error for now but neither  `eslint-disable` or `eslint-disable-line` seem to be working.  Any ideas about how to fix this?

**Comment by mdjermanovic** [comment, marked as answer]
`eslint-disable-*` doesn't affect parsing errors.

Can you check `parserOptions.ecmaVersion` in your config file? The `import()` syntax was introduced in ECMAScript 2020, so setting this value to 2020 or higher will probably fix the issue.

https://eslint.org/docs/latest/use/configure/language-options#specifying-parser-options

**Reply by jwerre** [reply]
I changed my `parserOptions.ecmaVersion` from 9 to 11 and the problem was solved. Thanks you @mdjermanovic
